src/lib/storage/run_length_column.cpp

```diff
@@ -36,15 +36,24 @@ std::shared_ptr<const pmr_vector<ChunkOffset>> RunLengthColumn<T>::end_positions
 template <typename T>
 const AllTypeVariant RunLengthColumn<T>::operator[](const ChunkOffset chunk_offset) const {
   PerformanceWarning("operator[] used");
+  const auto typed_value = get_typed_value(chunk_offset);
+  if (!typed_value.has_value()) {
+    return NULL_VALUE;
+  }
+  return *typed_value;
+}
```

Comment:
The doc for `operator[]` should have a comment telling people to use the new version whenever (?) they know the type.

Reply:
I added this to base_column.hpp, is that okay?
```
  // Returns the value at a given position. This is considerably slow since it returns an AllTypeVariant.
  // If you know the column type and its stored data type, consider using get_typed_value(chunk_offset) instead.
  virtual const AllTypeVariant operator[](const ChunkOffset chunk_offset) const = 0;
```